src/server/controllers/buzz.controller.ts

```diff
@@ -119,11 +126,75 @@ export function createBuzzTipTransactionHandler({
     if (fromAccountId === input.toAccountId)
       throw throwBadRequestError('You cannot send buzz to the same account');
 
-    return createBuzzTransaction({
+    const { entityType, entityId } = input;
+    let targetUserIds: number[] = [input.toAccountId].filter(isDefined);
```

Comment:
Doing a filter here seems like extra step
```suggestion
    let targetUserIds: number[] = input.toAccountId ? [input.toAccountId] : [];
```